docs/package.json

```
@@ -9,6 +9,7 @@
     "build": "docusaurus build",
     "build-api-docs": "python scripts/build-api-docs.py --with-r && tsx scripts/update-api-modules.ts",
     "build-api-docs:no-r": "python scripts/build-api-docs.py && tsx scripts/update-api-modules.ts",
+    "build-for-release": "python scripts/build-for-release.py",
```

```suggestion
    "build-all": "python scripts/build-for-release.py",
```

can we rename this to `build-all`?